packages/cli/src/index.ts

```diff
@@ -283,8 +298,20 @@ async function deployBot(medplum: MedplumClient, botConfig: MedplumBotConfig, bo
   }
 }
 
-function readBotConfig(botName: string): MedplumBotConfig | undefined {
-  return readConfig()?.bots?.find((b) => b.name === botName);
+function readBotConfigs(botName: string): MedplumBotConfig[] {
+  if (botName[0] === '*') {
```

Comment:
Can we support the wildcard anywhere in the string?

Reply:
Good idea, I'll add support for one wildcard for anywhere on the string.